jib-core/src/main/java/com/google/cloud/tools/jib/configuration/credentials/Credential.java

```diff
@@ -21,6 +21,9 @@
 // TODO: Move to lower-level package - probably at same level as Authorization.
 /** Holds credentials (username and password). */
 public class Credential {
+  // if the username is set to &lt;token&gt; the secret would be an refresh token, see details at
```

Comment:
FYI, this is internal code comment and not Javadoc, you don't want to use `&lt;` and such. It should look readable as code. Just say `<token>`.

Reply:
fixed